src/js/components/deployments/report.js

```diff
@@ -237,7 +237,12 @@ const mapStateToProps = state => {
   const { devices = {} } = state.deployments.byId[state.deployments.selectionState.selectedId] || {};
   const selectedDevices = state.deployments.selectedDeviceIds.map(deviceId => ({ ...state.devices.byId[deviceId], ...devices[deviceId] }));
   const deployment = state.deployments.byId[state.deployments.selectionState.selectedId] || {};
-  const { actor = {} } = state.organization.auditlog.events.find(event => event.object.id === state.deployments.selectionState.selectedId) || {};
+  // we can't filter by auditlog action via the api, so
```

Comment:
@mzedel if you need this filter, create a JIRA task and we'll implement it.

Reply:
maybe it would be easier to include the info in the deployment object (the step via auditlogs is more of a workaround IMO), if you don't mind I'll rather create a task for this...